docs/changes/cosmology/15488.api.rst

```diff
@@ -0,0 +1,2 @@
+Cosmology ``repr`` now uses ``repr`` for all the contained fields. For a prettier
+string representation use ``print`` instead.
```

Comment:
[`repr` is a Python built-in](https://docs.python.org/3/library/functions.html#repr). What a `Cosmology` has is a `__repr__()`.

Furthermore, `print()` prints the string representation, but that's not useful for anyone that wants to inspect or modify the string representation in their program, so that advice is incomplete.

Reply:
True. I've modified the changelog.